models/tags.js

```diff
@@ -0,0 +1,57 @@
+const db = require('../db');
+
+class tags {
+  static async createTag(entry_id,tag) {
+    // First have to find out if the tag already exist and if it does grab the tag id and make an insertion into journal_entries_tag table (the table takes an entry_id and a tag_id). If the tag doesn't exist it will first creat the tag and then do the function mentioned previously. 
+    const id = await db.query(`SELECT tags.name,tags.id FROM tags WHERE (tags.name = $1)`,[tag])
+     if (id.rows.length = 0){
+      await db.query(`INSERT INTO tags(name) VALUES($1)`,[tag])
+      await db.query(`SELECT tags.name,tags.id FROM tags WHERE (tags.name = $1)`,[tag])
```

Comment:
Why select something from table here if it's not being assigned to a variable for later use?

Reply:
making changes now